lib/ret_web/views/api/v1/avatar_view.ex

```diff
@@ -26,6 +27,7 @@ defmodule RetWeb.Api.V1.AvatarView do
       attributions: if(is_nil(avatar.attributions), do: [], else: avatar.attributions),
       allow_remixing: avatar.allow_remixing,
       allow_promotion: avatar.allow_promotion,
+      gltf_url: "#{RetWeb.Endpoint.url()}/api/v1/avatars/#{avatar.avatar_sid}/avatar.gltf?v=#{version}",
```

Comment:
thinking about it more, it seems like the right way to do this is to update things to encode the sid/version into the ETag for the response

Reply:
Adding ETAGS to this response (the avatar json) makes sense. Due to a few issues in our client (gltf caching, networking, etc) we expect the gltf url itself to change. I do think we should return a CDN url here so that it is cached.